ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -457,7 +457,10 @@ else
 	fi
 
 	# Check for flow sensor, launched as a background task to scan
-	px4flow start &
+	if param compare SENS_EN_PX4FLOW 1
+	then
+		px4flow start &
```

Comment:
Just curious, why do we start this in the background with &, when everything else starts regularly.

Reply:
It spends a fairly long time (~ 5 seconds) probing (multiple attempts) in the main.